TransformAST.cpp

```diff
@@ -0,0 +1,142 @@
+#include <string>
+
+#include <swift/AST/ASTContext.h>
+#include <swift/AST/ASTWalker.h>
+#include <swift/AST/Expr.h>
+#include <swift/AST/ParameterList.h>
+#include <swift/AST/Pattern.h>
+#include <swift/AST/Stmt.h>
+
+#include "TransformAST.h"
+
+void TransformFinalExpressionAndAddGlobal(swift::SourceFile &src_file)
+{
+    if(!src_file.Decls.empty())
```

Comment:
You could save a level of indentation by inverting the condition and doing an early return.